README.md

```diff
@@ -29,13 +29,13 @@ This dataset comprises 25 Hmong-Mien varieties, which were originally digitized
 
 - **Varieties:** 25
 - **Concepts:** 883
-- **Lexemes:** 21,573
+- **Lexemes:** 21,967
```

Comment:
Note that I switched to a splitting dataset, which explains the increase in words.